testdata/success_generics/types.go

```diff
@@ -0,0 +1,14 @@
+package success_generics
+
+//go:generate ../../go-builder-generator generate -f types.go -s Struct -d builders
+
+type GenericType[T any] struct {
```

Comment:
**thought :** It could be interesting to handle builders around generic types also